nextflow_schema.json

```diff
@@ -557,17 +598,29 @@
                     "fa_icon": "fas fa-percent",
                     "help_text": "This parameter specifies to use the calculated relative abundance (i.e., percentage of all hits). This is normally the default behaviour in mOTUs however we use counts by default in nf-core/taxprofiler for consistency with other classification/profiling tools.\n\n> Modifies tool parameter(s):\n> - mOTUs: `-c` (removed from the default nf-core/taxprofiler command)"
                 },
+                "motus_remove_ncbi_ids": {
+                    "type": "boolean",
+                    "description": "Turn on removing NCBI taxonomic IDs.",
+                    "fa_icon": "fas fa-address-card",
+                    "help_text": "By default mOTUs will report species names rather than NCBI Taxon IDs. In nf-core/taxprofiler we prefer taxon IDs due to interoperatbility and comparability with the output of other classifiers and profilers. If you prefer to have just species names, you can specify this to remove the IDs.\n\n> Modifies tool parameter(s):\n- mOTUs: `-p` (removed from the default nf-core/taxprofiler command)"
+                },
                 "motus_save_mgc_read_counts": {
                     "type": "boolean",
                     "description": "Turn on saving the mgc reads count.",
                     "fa_icon": "fas fa-save",
                     "help_text": "Turns on the saving of the read counts against each mOTU marker-gene clusters, in addition to per-taxon count/abundance reporting (in a separate file)\n\n> Modifies tool parameter(s):\n- mOTUs: `-M`"
                 },
-                "motus_remove_ncbi_ids": {
-                    "type": "boolean",
-                    "description": "Turn on removing NCBI taxonomic IDs.",
-                    "fa_icon": "fas fa-address-card",
-                    "help_text": "By default mOTUs will report species names rather than NCBI Taxon IDs. In nf-core/taxprofiler we prefer taxon IDs due to interoperatbility and comparability with the output of other classifiers and profilers. If you prefer to have just species names, you can specify this to remove the IDs.\n\n> Modifies tool parameter(s):\n- mOTUs: `-p` (removed from the default nf-core/taxprofiler command)"
+                "motus_longprep_splittinglength": {
+                    "type": "integer",
+                    "default": 300,
+                    "description": "Splitting length for the long reads.",
```

Comment:
Please add a `help_text` describing in more detail what/why it is doing

Reply:
Also missing the `'modifies XYZ tool parameters'`

Reply:
Updated PR